db/schema.rb

```diff
@@ -10,7 +10,7 @@
 #
 # It's strongly recommended that you check this file into your version control system.
 
-ActiveRecord::Schema.define(version: 20211109141544) do
+ActiveRecord::Schema.define(version: 20211117170147) do
```

Comment:
We also have Oracle and Postgresql schema

https://github.com/3scale/porta/tree/master/db